src/Package.cc

```diff
@@ -1963,6 +1964,53 @@ YCPValue PkgFunctions::GetSolverFlags()
     return ret;
 }
 
+/**
+ * @builtin SetAdditionalVendors
+ *
+ * @short set additional vendors which are compatible
+ * @description
+ * Select additional compatible vendors for installation.
```

Comment:
I think it's mainly used during upgrade...

```suggestion
 * Select additional compatible vendors for upgrade.
```